### PR TITLE
Fix assorted eslint errors to prepare for ci enforced lint

### PR DESCRIPTION
**Pull request by tsaok98**
Issue #71 Clean up lint errors and warnings

### CODE REVIEW

**Comment by cktang88** [comment]
I think we should keep the console logs, or replace with a logging library. Not logging really sucks

File: src/client/components/Buttons/LeftImgButton.tsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ interface Props extends ButtonProps {
 }
 
 const LeftImgButton = (props: Props): JSX.Element => {
-	const { img, imgAlt, color, text, onClick, ...ButtonProps } = props;
+	const { img, imgAlt, color, text, onClick, ...PropsButtonProps } = props;
```
Comment (cktang88) [inline src/client/components/Buttons/LeftImgButton.tsx]: This is just really bad naming

**Comment by bencooper222** [comment]
I don't know much about logging. @cktang88 do you want to investigate options, merge a PR into dev and then @tsaok98 can shift console logs to that library in this PR?

File: .eslintrc.js [hunk 2]
```diff
@@ -7,4 +7,8 @@ module.exports = {
 	rules: {
 		'sort-keys-fix/sort-keys-fix': ['error', 'asc', { caseSensitive: true, natural: true }],
 	},
+	env: {
+		browser: true,
```
Comment (bencooper222) [inline .eslintrc.js]: There are environment specific override eslint config files in the frontend and the server. Can you shift the environment declarations to each of those, respectively?

File: src/client/components/Buttons/LeftImgButton.tsx (hunk 1 above)
Comment (bencooper222) [reply src/client/components/Buttons/LeftImgButton.tsx]: ^

File: src/client/components/Containers/Collapsible.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 import React, { FunctionComponent, useRef, useState, useEffect } from 'react';
 import ResizeObserver from 'resize-observer-polyfill';
-import { FlexStartColumn, ContainerProps } from './FlexContainers';
+// import { FlexStartColumn, ContainerProps } from './FlexContainers';
```
Comment (bencooper222) [inline src/client/components/Containers/Collapsible.tsx]: Are all the commented out imports unnecessary? If so, delete them.

File: src/client/components/Sidebar/Sidebar.tsx [hunk 4]
```diff
@@ -41,12 +41,12 @@ const HorizontalLine = styled.hr`
...
 `;
 
-const HorizontalLineWithoutPad = styled.hr`
-	margin: 0 2rem;
-	width: calc(100% - 4rem);
-	border: 0.03125rem solid white;
-	opacity: 0.1;
-`;
+// const HorizontalLineWithoutPad = styled.hr`
+// 	margin: 0 2rem;
```
Comment (bencooper222) [inline src/client/components/Sidebar/Sidebar.tsx]: Ditto - if not needed, delete it.

**Comment by cktang88** [comment]
#90 for logging

File: src/client/components/Containers/Collapsible.tsx (hunk 3 above)
Comment (bencooper222) [reply src/client/components/Containers/Collapsible.tsx]: To be clear, not just in this file - in all changed ones (and other instances you notice). We use git: we don't need to save code.

**Comment by tsaok98** [comment]
Upon removing the first export, the compiler can no longer find the imports of each of the classes. However, it currently is set to use default exports. Thoughts on changes? The error appears when removing the normal export and replacing it with default export.

**Comment by bencooper222** [comment]
@tsaok98 note base branch change

**Comment by cktang88** [comment]
fix conflicts

**Comment by cktang88** [comment]
@tsaok98  please fix conflicts

**Comment by bencooper222** [comment]
I believe this is where I say that we're theoretically supposed to only rebase

**Comment by bencooper222** [comment]
Since this is branched off the wrong branch, I think the best course is

- [ ] remove merge commit
- [ ] cherry pick commits and drop them onto a new branch forked off of origin/dev 

I'm not good enough at git to know how to do this off the top of my head but if we could a) stash away the commits in this branch b) hard reset this branch to the current state of dev and then c) drop those commits back on, that would probably be best.

**Comment by cktang88** [comment]
What do you mean based off wrong branch @bencooper222 , I think this is based off dev?

**Comment by bencooper222** [comment]
No, it was originally branched off of ci-lint-setup then when I made that PR less monolithic, we decided to merge it into dev.

**Comment by cktang88** [comment]
@bencooper222 going to close this in favor of #154

**Pull request closed without merging**